### PR TITLE
Metavix/adding files

### PR DESCRIPTION
**Pull request by Metavix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        - [Connecting it to your computer over USB](https://docs.particle.io/guide/getting-started/connect/core/).
+    - If you are using a Particle Photon you have two ways to set it up:
+        - [Using your smart phone](https://docs.particle.io/guide/getting-started/start/Photon/).
+        - [Connecting it to your computer over USB](https://docs.particle.io/guide/getting-started/connect/Photon/).
+
+2. After claiming your Particle Device and setting up your Ubidots account, let's go to [Particle's Web IDE](https://build.particle.io/build).
+    * In the Particle's Web IDE create a new app and set the name.
+    * Go to the library tab.
+    * In contributed library write Ubidots and select the Ubidots library.
+    * Click on **INCLUDE IN APP**. And return to "MYAPP.ino"
```
Comment (woakas) [inline README.md:27]: @Metavix should be MyApp.ino

File: README.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+To subscribe to a variable, you need to put the data source label where is your variable, and the variable label in the code, and the data value will save in the "message" variable of callback function.
+
+```cpp
+// This #include statement was automatically added by the Particle IDE.
+#include "MQTT/MQTT.h"
+
+// This #include statement was automatically added by the Particle IDE.
+#include "UbidotsMQTT.h"
+#define TOKEN "Your_Token_Here"  // Add here your Ubidots TOKEN
+#define DATA_SOURCE_IDENTEFIER "23002e000551343530xxxxx" // Add the data source label of Ubidots Cloud
```
Comment (woakas) [inline README.md]: Change for **MyParticle.**

Comment (Metavix) [inline README.md]: Ok  i will do that

File: README.md (hunk 1 above)
Comment (Metavix) [inline README.md:27]: It is like this in Particle Cloud

**Pull request merged**